index.html

```diff
@@ -8,7 +8,17 @@
     <link rel="stylesheet" href="index.css">
 </head>
 <body>
-    <h1>Welcome</h1>
+    <h2>NS</h2>
+
+    <div class="container">
+        <nav>
+        <ul>
+            <li><a href="#">Welcome</a></li>
+            <li><a href="#">Gallery</a></li>
+        </ul>
+       </nav>
+
+    </div>
```

Comment:
Good job implementing the basic nav structure 💯

Reply:
Thank you 🤝